src/audio-worklet/free-queue/examples/simple-passthrough/main.js

```diff
@@ -0,0 +1,79 @@
+import FreeQueue from "../../src/free-queue.js"
+import { QUEUE_SIZE } from "./constants.js";
+
+const toogleButton = document.getElementById("toogle");
+toogleButton.disabled = false;
+
+// Create 2 FreeQueue instances with 4096 buffer length and 1 channel.
+const inputQueue = new FreeQueue(QUEUE_SIZE, 1);
+const outputQueue = new FreeQueue(QUEUE_SIZE, 1);
+// Create an atomic state for synchronization between worker and AudioWorklet.
+const atomicState = new Int32Array(
+    new SharedArrayBuffer(1 * Int32Array.BYTES_PER_ELEMENT)
+);
+
+let audioContext;
+let playing = false;
+
+/**
+ * Function to create and initialize AudioContext.
+ * @returns {Promise<AudioContext>}
+ */
+const createAudioContext = async () => {
+  const audioContext = new AudioContext();
+  await audioContext.audioWorklet.addModule("basic-processor.js");
+  const oscillator = new OscillatorNode(audioContext);
+  const processorNode =
+      new AudioWorkletNode(audioContext, 'basic-processor', {
+          processorOptions: {
+              inputQueue,
+              outputQueue,
+              atomicState
+          }
+      });
+  oscillator.connect(processorNode).connect(audioContext.destination);
+  audioContext.suspend();
+  oscillator.start();
+  console.log("AudioContext created.");
+  return audioContext;
+}
```

Comment:
So I think coi-serviceworker.js is causing problems in case of running format script.
How should I proceed with that?

Is there any way to exclude files?

Reply:
There is also issue with emscripten genereated files

Reply:
There's a way to exclude those files from linting. See:
https://github.com/GoogleChromeLabs/web-audio-samples/blob/main/.eslintrc.json#L10